oci/layer/tar_extract.go

```diff
@@ -136,7 +144,13 @@ func (te *TarExtractor) restoreMetadata(path string, hdr *tar.Header) error {
 		if errors.Cause(err) != unix.ENOTSUP {
 			return errors.Wrapf(err, "clear xattr metadata: %s", path)
 		}
-		log.Warnf("xattr{%s} ignoring ENOTSUP on clearxattrs", path)
+		if !te.enotsupWarned {
+			log.Warnf("xattr{%s} ignoring ENOTSUP on clearxattrs", path)
+			log.Warnf("xattr{%s} destination filesystem does not support xattrs, further warnings will be suppressed", path)
+			te.enotsupWarned = true
+		} else {
```

Comment:
Is it sufficient to drop a `Debugf` into an `else` for the repeated warnings as done here - or would you prefer everything to come out as a warning (original behavior) if the loglevel is debug?